packages/optimizer/lib/DomTransformer.js

```diff
@@ -20,6 +20,8 @@ const log = require('./log');
 let fetch = require('node-fetch');
 const RuntimeVersion = require('@ampproject/toolbox-runtime-version/lib/RuntimeVersion');
 const fetchRuntimeParameters = require('./fetchRuntimeParameters');
+const cache = require('./cache.js');
+const {OperationCanceledException} = require('typescript');
```

Comment:
is `OperationCanceledException` unused? If so, shouldn't this have been caught by prettier or something?